diff --git a/api/opensource/mfx_dispatch/CMakeLists.txt b/api/opensource/mfx_dispatch/CMakeLists.txt
index 58d69b5..9a83086 100644
--- a/api/opensource/mfx_dispatch/CMakeLists.txt
+++ b/api/opensource/mfx_dispatch/CMakeLists.txt
@@ -85,14 +85,6 @@ else( )
   endif( )
 
   set(no_warnings "-Wno-unknown-pragmas -Wno-unused")
-
-  set(CMAKE_CXX_FLAGS "-pipe -fPIC")
-  set(CMAKE_CXX_FLAGS_DEBUG   "-O0 -Wall ${no_warnings} -Wformat -Wformat-security -g -D_DEBUG")
-  set(CMAKE_CXX_FLAGS_RELEASE "-O2 -D_FORTIFY_SOURCE=2 -fstack-protector -Wall ${no_warnings} -Wformat -Wformat-security  -DNDEBUG")
-
-  if (DEFINED CMAKE_FIND_ROOT_PATH)
-    append("--sysroot=${CMAKE_FIND_ROOT_PATH} " LINK_FLAGS)
-  endif (DEFINED CMAKE_FIND_ROOT_PATH)
 endif( )
 
 include_directories (
