diff --git a/api/opensource/mfx_dispatch/pkg-config.pc.cmake b/api/opensource/mfx_dispatch/pkg-config.pc.cmake
index 2adbcac..811d4ab 100644
--- a/api/opensource/mfx_dispatch/pkg-config.pc.cmake
+++ b/api/opensource/mfx_dispatch/pkg-config.pc.cmake
@@ -5,5 +5,5 @@ Version: @MFX_VERSION_MAJOR@.@MFX_VERSION_MINOR@
 prefix=@CMAKE_INSTALL_PREFIX@
 libdir=@CMAKE_INSTALL_FULL_LIBDIR@
 includedir=@CMAKE_INSTALL_FULL_INCLUDEDIR@
-Libs: -L${libdir} -lmfx
+Libs: -L${libdir} -lmfx -lstdc++ -ldl
 Cflags: -I${includedir}
