diff --git a/cmake/FindMediaSDK.cmake b/cmake/FindMediaSDK.cmake
index ca54e1e..7ce286b 100644
--- a/cmake/FindMediaSDK.cmake
+++ b/cmake/FindMediaSDK.cmake
@@ -17,7 +17,7 @@ function(FindMediaSDK)
       message(FATAL_ERROR "No MediaSDK distribution is found.")
     endif()
     set(MFX_INCLUDES "${MFX_HOME}/include")
-    set(MFX_LIBRARY  "${MFX_HOME}/lib/lin_x64/libmfx.a")
+    set(MFX_LIBRARY  "${MFX_HOME}/lib/libmfx.a")
     message(STATUS "MediaSDK include directory: ${MFX_INCLUDES}")
     message( STATUS "MediaSDK library directory: ${MFX_LIBRARY}")
     include_directories(${MFX_INCLUDES})
diff --git a/meson.build b/meson.build
index f6bfc08..9cb4e68 100644
--- a/meson.build
+++ b/meson.build
@@ -35,7 +35,7 @@ mfx_home = get_option ('MFX_HOME')
 mfx_inc = include_directories(['.', 'gst/mfx', 'gst-libs/mfx', 'parsers', '@0@/include'.format(mfx_home)])
 
 compiler = meson.get_compiler('c')
-libmfx = compiler.find_library('mfx', dirs: '@0@/lib/lin_x64/'.format(mfx_home))
+libmfx = compiler.find_library('mfx', dirs: '@0@/lib/'.format(mfx_home))
 mfx_deps += [libmfx]
 
 libstdcc = compiler.find_library('stdc++')
