diff --git a/cmake/FindMediaSDK.cmake b/cmake/FindMediaSDK.cmake
index ca54e1e..6886b38 100644
--- a/cmake/FindMediaSDK.cmake
+++ b/cmake/FindMediaSDK.cmake
@@ -1,5 +1,5 @@
 function(FindMediaSDK)
-  pkg_check_modules(MFX  QUIET  mfx)
+  #pkg_check_modules(MFX  QUIET  mfx)
   set(MFX_LIBS "")
   if(MFX_FOUND)
      message(STATUS "MFX_LIBRARY_DIRS: ${MFX_LIBRARY_DIRS}")
@@ -17,7 +17,7 @@ function(FindMediaSDK)
       message(FATAL_ERROR "No MediaSDK distribution is found.")
     endif()
     set(MFX_INCLUDES "${MFX_HOME}/include")
-    set(MFX_LIBRARY  "${MFX_HOME}/lib/lin_x64/libmfx.a")
+    set(MFX_LIBRARY  "${MFX_HOME}/lib/libmfx.a")
     message(STATUS "MediaSDK include directory: ${MFX_INCLUDES}")
     message( STATUS "MediaSDK library directory: ${MFX_LIBRARY}")
     include_directories(${MFX_INCLUDES})
