From 0a543d16962919abf8c58aa6e6c4a7836b51d247 Mon Sep 17 00:00:00 2001
From: Jacek Danecki <jacek.danecki@intel.com>
Date: Wed, 9 Dec 2015 14:25:27 +0100
Subject: [PATCH 076/153] Add device ID for SKL ULT GT3

---
 include/drm/i915_pciids.h |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/include/drm/i915_pciids.h b/include/drm/i915_pciids.h
index 9c74721..a05e6d7 100644
--- a/include/drm/i915_pciids.h
+++ b/include/drm/i915_pciids.h
@@ -278,6 +278,7 @@
 
 #define INTEL_SKL_GT3_IDS(info) \
 	INTEL_VGA_DEVICE(0x1926, info), /* ULT GT3 */ \
+	INTEL_VGA_DEVICE(0x1927, info), /* ULT GT3 */ \
 	INTEL_VGA_DEVICE(0x192B, info), /* Halo GT3 */ \
 	INTEL_VGA_DEVICE(0x192A, info)  /* SRV GT3 */
 
-- 
1.7.1

