From 2ea4c2d3b43326e270f5815ea2400f59225571cb Mon Sep 17 00:00:00 2001
From: Hong Liu <hong.liu@intel.com>
Date: Mon, 8 Aug 2016 17:27:27 +0800
Subject: [PATCH 143/153] [VPG]: drm/i915: add more missing SKL GT2 pci ids

Signed-off-by: Hong Liu <hong.liu@intel.com>
---
 include/drm/i915_pciids.h |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/include/drm/i915_pciids.h b/include/drm/i915_pciids.h
index 7a1d003..77effb2 100644
--- a/include/drm/i915_pciids.h
+++ b/include/drm/i915_pciids.h
@@ -271,8 +271,10 @@
 	INTEL_VGA_DEVICE(0x1913, info), /* ULT GT2 */ \
 	INTEL_VGA_DEVICE(0x1916, info), /* ULT GT2 */ \
 	INTEL_VGA_DEVICE(0x1921, info), /* ULT GT2F */ \
+	INTEL_VGA_DEVICE(0x1915, info), /* ULX GT2 */ \
 	INTEL_VGA_DEVICE(0x191E, info), /* ULX GT2 */ \
 	INTEL_VGA_DEVICE(0x1912, info), /* DT  GT2 */ \
+	INTEL_VGA_DEVICE(0x1917, info), /* DT  GT2 */ \
 	INTEL_VGA_DEVICE(0x191B, info), /* Halo GT2 */ \
 	INTEL_VGA_DEVICE(0x191A, info), /* SRV GT2 */ \
 	INTEL_VGA_DEVICE(0x191D, info)  /* WKS GT2 */
-- 
1.7.1

