diff --git a/media_driver/linux/ult/CMakeLists.txt b/media_driver/linux/ult/CMakeLists.txt
index 2fac051..ae5769a 100644
--- a/media_driver/linux/ult/CMakeLists.txt
+++ b/media_driver/linux/ult/CMakeLists.txt
@@ -42,13 +42,3 @@ else ()
         include_directories(BEFORE ${LIBVA_INCLUDE_DIRS})
     endif ()
 endif ()
-
-add_subdirectory(libdrm_mock)
-add_subdirectory(ult_app)
-
-enable_testing()
-add_test(NAME test_devult COMMAND devult ${UMD_PATH})
-set_tests_properties(test_devult
-    PROPERTIES PASS_REGULAR_EXPRESSION "PASS")
-set_tests_properties(test_devult
-    PROPERTIES FAIL_REGULAR_EXPRESSION "FAIL")
