diff --git a/Source/GmmLib/ULT/CMakeLists.txt b/Source/GmmLib/ULT/CMakeLists.txt
index 10ab4ea..63505c2 100644
--- a/Source/GmmLib/ULT/CMakeLists.txt
+++ b/Source/GmmLib/ULT/CMakeLists.txt
@@ -122,14 +122,4 @@ else()
 	)
  endif()
 
-add_custom_target(Run_ULT ALL DEPENDS GMMULT)
-
-add_custom_command(
-      TARGET Run_ULT
-      POST_BUILD
-      COMMAND echo running ULTs
-      COMMAND ${EXE_NAME}  --gtest_filter=CTest*
-    )
-
-
 
